src/components/Gesture.svelte

```diff
@@ -169,26 +171,8 @@
     removeRecording(gesture.getId(), recording.ID);
   }
 
-  // Selecting this gesture for recording. Updates settings accordingly
-  // If gesture is already selected, the selection is removed.
-  // If bluetooth is not connected, open connection prompt by calling callback
-  function selectClicked(): void {
-    if (!$state.isInputConnected) {
-      chosenGesture.update(gesture => {
-        gesture = null;
-        return gesture;
-      });
-      onNoMicrobitSelect();
-      return;
-    }
```

Comment:
Removing `onNoMicrobitSelect` looks good to me! The record button is disabled when `!$state.isInputConnected`, so I don't think this part of the function can be ran at all. In short, `onNoMicrobitSelect` is not being called.